app/javascript/chat/ChatChannelSettings/ActiveMembershipsSection.jsx

```diff
@@ -19,25 +20,23 @@ const ActiveMembershipSection = ({
       data-active-count={activeMemberships ? activeMemberships.length : 0}
     >
       <h3 className="mb-2 active_members">Members</h3>
-      {activeMembershipList && activeMembershipList.length > 0
-        ? activeMembershipList.map((activeMembership) => (
-            <Membership
-              membership={activeMembership}
-              removeMembership={removeMembership}
-              membershipType="active"
-              currentMembershipRole={currentMembershipRole}
-              className="active-member"
-            />
-          ))
-        : null}
+      {activeMembershipList.map((activeMembership) => (
+        <Membership
+          membership={activeMembership}
+          removeMembership={removeMembership}
+          membershipType="active"
+          currentMembershipRole={currentMembershipRole}
+          className="active-member"
+        />
+      ))}
       <div className="row align-center">
-        <button
+        <Button
```

Comment:
🥳 Thanks for changing to the design system button. Just FYI, you don't need to include the following CSS classes, `crayons-btn` and `crayons-btn-s`. Internally, the `<Button />` component adds the `crayons-btn` class and for the `crayons-btn-s`, just add the prop/value `size='s'`.